.gitattributes

```diff
@@ -1,7 +1,16 @@
 # Force the bash scripts to checkout with Unix line endings, even on Windows
-
 build-* eol=lf
 config eol=lf
 run-* eol=lf
 spec_* eol=lf
 translator-tests eol=lf
+
+# Explicitly mark binary files for Git
+*.bin binary
+
+# Highlight ksy and kst files as YAML on GitHub
+*.kst linguist-language=yaml
+*.ksy linguist-language=yaml
```

Comment:
That's actually a good point if we want to start a process registering KSY as legitimate language in GH linguist?

Reply:
Good idea! It looks like we meet the requirement of "hundreds of repos" using the language (there are 256 GitHub repos with KSY files in them), so I've opened a PR on the Linguist repo: github/linguist#4830